spec/views/user_index_spec.rb

```diff
@@ -0,0 +1,52 @@
+require 'rails_helper'
+
+RSpec.feature 'User Index Page', type: :feature do
+  let(:user1) { User.create(name: 'User 1', photo: 'user1.jpg') }
+  let(:user2) { User.create(name: 'User 2', photo: 'user2.jpg') }
+
+  scenario 'viewing the User Index Page' do
+    # Create some sample users
+    user1
+    user2
+
+    visit users_path
+
+    # Verify that the username of all other users is displayed
+    expect(page).to have_content('User 1')
+    expect(page).to have_content('User 2')
+
+    # Verify that the profile picture for each user is displayed
+    expect(page).to have_css("img[src*='user1.jpg']")
+    expect(page).to have_css("img[src*='user2.jpg']")
+
+    # Verify that the number of posts each user has written is displayed
+    # expect(page).to have_content('User 1 has written 0 posts.')
+    # expect(page).to have_content('User 2 has written 0 posts.')
```

Comment:
- [ ] Please remove any commented-out code it makes your codebase look unorganised and harder to read.